### PR TITLE
Update portal.properties.EXAMPLE and reference docs to document installation map url

### LINKED ISSUE #8055: Document configuration for installation map url
The `portal.properties.EXAMPLE` should be updated to include configuration for the installation map and the docs should be updated accordingly.

### PR DESCRIPTION
**Pull request by jtquach1**
Fix #8055

Describe changes proposed in this pull request:
- Added commented `installation_map_url` configuration to portal.properties.EXAMPLE
- Updated docs and created section on `installation_map_url`

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [Giphy CAPTURE](https://giphy.com/apps/giphycapture) or [Peek](https://github.com/phw/peek)
No screenshots/gifs

### CODE REVIEW

**Comment by jtquach1** [comment]
My apologies, I didn't check that `installation_map_url` was already added from `master`. Anyhow, only the portal.properties.EXAMPLE has been modified.

File: src/main/resources/portal.properties.EXAMPLE [hunk 1]
```diff
@@ -327,3 +327,6 @@ ehcache.cache_type=none
 
 # Enable gsva to query
 # skin.show_gsva=true
+
+# Installation map URL for public site
```
Comment (Luke-Sikina) [inline src/main/resources/portal.properties.EXAMPLE:331]: This is a bit misleading - private portals could create their own installation map if they wanted. How about 

>Installation map URL for installation map iframes. If blank or commented out, no map will be shown and that page will be hidden.`

**Review by Luke-Sikina — CHANGES_REQUESTED** [review]
This is missing the updates to `cbioportal/docs/portal.properties-Reference.md`. Once you add those lmk.

**Comment by jtquach1** [comment]
> This is missing the updates to `cbioportal/docs/portal.properties-Reference.md`. Once you add those lmk.

Done!

File: docs/portal.properties-Reference.md [hunk 2]
```diff
@@ -170,6 +170,13 @@ By default, the initial x-axis limit for a survival plot is the time of the late
 survival.initial_x_axis_limit=
 ```
 
+## Display installation map
+This setting specifies the URL for the iframes of a given portal's installation map on the homepage and standalone page. By default, the installation map configuration is commented out with the iframes and standalone page hidden. 
+The configuration below is used for https://www.cbioportal.org/.
```
Comment (Luke-Sikina) [inline docs/portal.properties-Reference.md:175]: This is good, but let's add something about how they can set up their own installation map instance. Something like:
> The source code for the installation map can be found [here](https://github.com/cbioportal/installation-map).

**Review by Luke-Sikina — APPROVED** [review]

File: docs/portal.properties-Reference.md (hunk 2 above)
Comment (jtquach1) [reply docs/portal.properties-Reference.md:175]: I should update the repo readme and script names sometime soon. 😅

**Pull request merged**